pkg/util/nifi/common.go

```diff
@@ -15,8 +15,8 @@ const (
 	NodeNameTemplate       = PrefixNodeNameTemplate + RootNodeNameTemplate + SuffixNodeNameTemplate
 
 	// TimeStampLayout defines the date format used.
-	TimeStampLayout     = "Mon, 2 Jan 2006 15:04:05 GMT"
-	NifiDataVolumeMount = "nifi-data"
+	TimeStampLayout        = "Mon, 2 Jan 2006 15:04:05 GMT"
+	NifiDataVolumeMountKey = "nifi-data"
```

Comment:
It might be safer to pick a label that's named after the project, to prevent user re-use? Similar to how k8s use `app.kubernetes.io/name` as an example.
Something like `nifikop.konpyutaika.github.io/storage: managed`?

Reply:
`nifi.konpyutaika.com` as the group version of the k8s resources could be better no ?

Reply:
> `nifi.konpyutaika.com` as the group version of the k8s resources could be better no? And you could do something like `var labelNifiData string = fmt.Sprintf("%s/nifi-data", v1alpha1.GroupVersion.Group)`

My only worry with `nifi.konpyutaika.com` is that we don't own the domain name? Or maybe that doesn't matter, and it's a technicality?

Reply:
We are already using it as k8s Group 😊

Reply:
I hadn't looked closely enough at the CRD group names! We should definitely keep them consistent :)
I agree with your `fmt.Sprintf` suggestion above :D